_posts/wiki/2019-09-03-Undergrad-Guide.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Getting Started as an Undergraduate in the HIRO Group
+description: Answering the questions you didn't know to ask
+permalink: join_us.html
+---
+
+# Contents
+This guide is intended for new undergraduate members with limited exposure to
+ROS and other tools used in the HIRO Group. This page in conjunction with the
+new member to-do list page will help you through setting up all of the software
```

Comment:
There are ways to reference other pages within Jekyll, you should do it here! See line 16 of https://github.com/HIRO-group/HIRO-group.github.io/blob/master/_posts/wiki/2018-06-14-ROS-indigo-installation.md for an example